DKP2_Developers/PDM/Основное/Подсистема.yaml

```
@@ -0,0 +1,2 @@
+Интерфейс:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)